apps/web/src/pages/Settings.tsx

```
@@ -0,0 +1,77 @@
+
+      {/* Tab bar */}
+      <div class="flex gap-1 border-b border-border px-6">
+        {tabs.map((tab) => (
+          <button
+            class={`px-4 py-2.5 text-sm font-medium transition-colors ${
+              activeTab() === tab.id
+                ? "border-b-2 border-primary text-foreground"
+                : "text-muted-foreground hover:text-foreground"
+            }`}
+            onClick={() => setActiveTab(tab.id)}
+          >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)